.github/workflows/build-release-zip.yml

```diff
@@ -0,0 +1,55 @@
+name: Build release zip
+
+on:
+  workflow_dispatch:
+  push:
+   branches:
+    - trunk
+
+jobs:
+  build:
+    name: Build release zip
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Cache vendor
+        id: cache-composer
+        uses: actions/cache@v3
+        env:
+          cache-name: cache-vendor
+        with:
+          path: |
+            vendor
+            ~/.composer/cache
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/composer.json') }}
+
```

Comment:
Composer only uses dev dependencies and isn't loaded in production (ie, autoload isn't referenced) so can be ignored from the build here. It's not included in the wp.org version either, see https://plugins.trac.wordpress.org/browser/simple-local-avatars/trunk

Reply:
The composer dependencies install step is removed.